content/sensu-go/5.0

```diff
@@ -0,0 +1 @@
+../../archived/sensu-go/5.0
```

Comment:
@hillaryfraley we don't want to commit these symlinks, just create them locally on an as-needed basis (e.g. create symlink so we can generate PDF of archived docs version from server running on local laptop)

Reply:
Ahhhh ok so we stop and commit after `Edit config.toml to remove the version being archived`?